2.2/Dockerfile

```diff
@@ -14,6 +14,10 @@ LABEL io.k8s.description="Platform for building and running Ruby 2.2 application
       io.openshift.expose-services="8080:http" \
       io.openshift.tags="builder,ruby,ruby22"
 
+ONBUILD ADD . /tmp/src
+ONBUILD RUN $STI_SCRIPTS_PATH/assemble
+ONBUILD CMD $STI_SCRIPTS_PATH/run
```

Comment:
oops.  no it was not.  and there's a problem w/ doing it anyway which is that it makes it hard for users to extend our images with their own layers, since there's no way to avoid having those commands executed.